### ISSUE TITLE
Validate Ethereum connection and signing against testnet

### ISSUE THREAD

**Issue by ethanfrey** [iov-one/iov-core#411]
After #410 please manually verify this works properly with Kovan, Rinkeby, and Ropstein.

Note that the high block times (30s on Ropstein) may raise up some issues in the implementationAPI

**Comment by ethanfrey** [comment]
Already done by @LucasIsasmendi
